fooof/synth/transform.py

```diff
@@ -14,33 +16,124 @@ def rotate_spectrum(freqs, power_spectrum, delta_f, f_rotation):
         Frequency axis of input power spectrum, in Hz.
     power_spectrum : 1d array
         Power values of the spectrum that is to be rotated.
-    delta_f : float
-        Change in power law exponent to be applied.
-        Positive is counterclockwise rotation (flatten).
-        Negative is clockwise rotation (steepen).
+    delta : float
+        Change in aperiodic exponent to be applied.
+        Positive is clockwise rotation (steepen).
+        Negative is counterclockwise rotation (flatten).
     f_rotation : float
-        Frequency (Hz) at which to do the rotation, such that power at that frequency is unchanged.
+        Frequency value, in Hz, about which rotation is applied, at which power is unchanged.
 
     Returns
     -------
     rotated_spectrum : 1d array
         Rotated power spectrum.
     """
 
-    # Check that the requested frequency rotation value is within the given range
-    if f_rotation < freqs.min() or f_rotation > freqs.max():
-        raise ValueError('Rotation frequency not within frequency range.')
+    mask = (np.abs(freqs) / f_rotation)**-delta
+    rotated_spectrum = mask * power_spectrum
+
+    return rotated_spectrum
+
+
+def translate_spectrum(power_spectrum, delta):
+    """Translate a spectrum, changing the offset value.
+
+    Parameters
+    ----------
+    power_spectrum : 1d array
+        Power values of the spectrum that is to be translated.
+    delta : float
```

Comment:
yeah, especially since there are multiple functions, where delta is used and means different things, it'd be easier to save the user from having to interpret what delta means from the context of the function.